includes/wikia/services/tests/SassServiceTest.php

```diff
@@ -25,7 +25,7 @@ public function testCompileInlineCss() {
 	}
 }
 CSS;
-		$sass = SassService::newFromString( $css );
+		$sass = SassService::newFromString( $css, 0, '' );
 		$sass->setSassVariables( self::$sassVariables );
 
 		$result = $sass->getCss( false ); # no cache
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)